pom.xml

```diff
@@ -42,6 +42,8 @@
 
     <properties>
         <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
+1 for enforcing FindBugs. If @rsandell wants to get a more graceful behavior, the best option would be to tweak profiles in the new parent POM